.travis.yml

```diff
@@ -29,6 +29,8 @@ before_install:
   # Install dependencies of facter
   - wget https://s3.amazonaws.com/kylo-pl-bucket/yaml-cpp-0.5.1_install.tar.bz2
   - tar xjvf yaml-cpp-0.5.1_install.tar.bz2 --strip 1 -C $USERDIR
+  - wget https://github.com/puppetlabs/leatherman/releases/download/${LEATHERMAN_VERSION}/leatherman.tar.gz
```

Comment:
Also, we probably don't want to use /latest/ - we may want to keep stable branches of projects on a z-release bump for bugfixes in the future (even though we havne't yet done a leatherman z)